src/components/20-molecules/datepicker/index.js

```diff
@@ -550,32 +550,50 @@ class AXADatepicker extends NoShadowDOM {
       }
     }
 
-    if (this.allowedyears && !this.allowedyears.includes(this.year)) {
-      // to avoid that datepicker don't show any days to select and looks like "empty"
+    // does the range of allowed years exclude the starting year?
+    if (
+      this.allowedyears &&
+      this.year &&
+      !this.allowedyears.includes(this.year)
+    ) {
+      // yes, so in order to prevent the datepicker from showing an "empty" month sheet
+      // without any days to select, force-pick the starting year as the first of the range of allowed years
       const [newStartYear] = this.allowedyears;
       this.year = newStartYear;
     }
 
     const { year, month, day, allowedyears, locale, startDate } = this;
 
-    this._date = overrideDate(year, month, day, startDate);
+    const status = {};
+    this._date = overrideDate(year, month, day, startDate, status);
     const { _date } = this;
+    // did the start date get actually overridden via one or more of the
+    // date parts (year, month, day)?
+    if (status.set) {
+      // yes, so the resulting _date is application-selected
+      delete _date.type;
```

Comment:
Is it possible to not delete this prop `type` to have more debug informations?

Reply:
It would be incorrect, since `_date.type === 'default'` is no longer true. 

This fact can itself serve as debug info...